janhq/jan
# feat: temporary remove dark mode :(

## Describe Your Changes

this is temporary remove dark mode for help me create theme, we will avoid using tailwind classname inline with prefix `dark:xxxx` for support theme

## Fixes Issues

#2088 


## Test scenario
- Make sure all element on light mode are good
- Remove My Setting  / Appearance menu 

![Screenshot 2024-02-27 at 14 30 44](https://github.com/janhq/jan/assets/10354610/f55978d7-7ddc-4727-a1ec-994c55dadc5f)


## Self Checklist
